File: app/src/main/java/com/kaltura/kflow/ui/debug/DebugListener.java
package com.kaltura.kflow.ui.debug;

import org.json.JSONObject;

/**
 * Created by alex_lytvynenko on 20.11.2018.
 */
public interface DebugListener {
    void setRequestInfo(String url, String method, int code);

    void setRequestBody(JSONObject requestBody);

    void setResponseBody(JSONObject responseBody);

    void onError();
}

File: app/src/main/java/com/kaltura/kflow/presentation/debug/DebugListener.kt
package com.kaltura.kflow.presentation.debug

import org.json.JSONObject

/**
 * Created by alex_lytvynenko on 20.11.2018.
 */
interface DebugListener {
    fun setRequestInfo(url: String, method: String, code: Int)
    fun setRequestBody(requestBody: JSONObject)
    fun setResponseBody(responseBody: JSONObject)
    fun onError()
}